### ISSUE TITLE
Draw markings

### ISSUE THREAD

**Issue by paulknorr** [tjeerdintveen/Vurig-Calendar#14]
It is possible to draw markings with characters?

**Comment by DKalachniuk** [comment]
Yes it is possible. You can do what ever you need there. method -> drawRect: . Find comment line //Draw markings
Change it like yuo want
